.changes/next-release/feature-AwsCrtHttpClientforJavav2-c8d7d24.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "feature",
+    "category": "Aws Crt Http Client for Java v2",
+    "contributor": "",
+    "description": "Exposed the connectionAcquisitionTimeout field in AwsCrtHttpClient and AwsCrtAsyncHttpClient and pass it through"
```

Comment:
`pass it through` is implementation detail and can be omitted. 


Suggesting the following:

Allow users to configure `connectionAcquisitionTimeout` on `AwsCrtHttpClient` and `AwsCrtAsyncHttpClient`.